### ISSUE TITLE
500 Error when trying to checkout with PayPal Express

### ISSUE THREAD

**Issue by marpstar** [angelleye/paypal-woocommerce#684]
Upgraded from 1.3.1 to 1.4.2 tonight and am now getting a 500 error when trying to process payments via PayPal Express. PayFlow payments appear to be unaffected. Any ideas?

**Comment by angelleye** [comment]
So you're using the release branch from GitHub, or did you apply the auto-update which is 1.4.1?

500 Error means a PHP failure is occurring, so you'd need to look at the PHP error log to get more details on that please.

**Comment by marpstar** [comment]
```
[05-May-2017 03:11:19 UTC] PHP Parse error:  syntax error, unexpected '&&' (T_BOOLEAN_AND), expecting ')' in /home/oneupusallc/public_html/wp-content/plugins/paypal-for-woocommerce/angelleye-includes/express-checkout/class-wc-gateway-paypal-express-request-angelleye.php on line 984
```

**Comment by marpstar** [comment]
and you're right, I'm running 1.4.1

**Comment by kcppdevelopers** [comment]
#674

**Comment by angelleye** [comment]
@marpstar This is fixed in the release branch and will be included in the 1.4.2 update coming soon.

**Comment by marpstar** [comment]
Looks good @angelleye. Thank you!!
